src/contrib/parallel-rdp/parallel-rdp-standalone/vulkan/wsi.hpp

```diff
@@ -210,7 +210,7 @@ class WSI
 
 	// The simple approach. WSI internally creates the context with instance + device.
 	// Required information about extensions etc, is pulled from the platform.
-	bool init_context_from_platform(unsigned num_thread_indices, const Context::SystemHandles &system_handles);
+	bool init_context_from_platform(InstanceFactory*, unsigned num_thread_indices, const Context::SystemHandles &system_handles);
```

Comment:
This is from my fork of parallel-rdp which I regularly keep updated. Let me know if you strictly want the upstream version instead. I can work on talking to Themaister